app/src/main/java/org/schabi/newpipe/util/StreamDialogEntry.java

```diff
@@ -203,4 +209,43 @@ private static void openChannelFragment(final Fragment fragment,
                 fragment.requireActivity().getSupportFragmentManager(),
                 item.getServiceId(), uploaderUrl, item.getUploaderName());
     }
+
+    /////////////////////////////////////////////
+    // helper functions                        //
+    /////////////////////////////////////////////
+
+    private interface InfoCallback {
+        void onInfo(SinglePlayQueue item);
+    }
+
```

Comment:
Sorry for not noticing this before... instead of creating a new interface just for that one method you can use [a Java `Consumer<SinglePlayQueue>`](https://docs.oracle.com/javase/8/docs/api/java/util/function/Consumer.html). Just replace `InfoCallback` with `Consumer<SinglePlayQueue>` and replace `callback.onInfo` with `callback.accept`